NEWS

```diff
@@ -23,6 +23,7 @@ PHP                                                                        NEWS
 
 - GMP:
   . The GMP class is now final and cannot be extended anymore. (Girgias)
+  . RFC: Change GMP bool cast behavior (Saki Takamachi)
```

Comment:
```suggestion
  . RFC: Change GMP bool cast behavior. (Saki Takamachi)
```
There needs to be a `.` before the author for the NEWS parser to correctly parse the line.

Reply:
Thanks!